vendor/assets/stylesheets/ombulabs/styleguide/4-molecules/_code.scss

```diff
@@ -0,0 +1,15 @@
+@mixin code {
+  background-color: $lighter-grey;
+  -webkit-box-decoration-break: clone;
+  box-decoration-break: clone;
+  padding: 0 10px;
+  text-align: left;
+}
+code{
+  @include code;
+}
+pre{
+  code{
+    @include code;
+  }
```

Comment:
@nykka This looks like a bit of over-engineering, why not just use a simple CSS rule?

For example: 

```
code, pre code {
  background-color: $lighter-grey;
  // ... 
}
```

Reply:
Makes sense!